ci/ray_ci/clean_up_old_docker_commit_tags.py

```diff
@@ -0,0 +1,193 @@
+import requests
+import json
+from datetime import datetime, timezone, timedelta
+import re
+import subprocess
+from typing import Set
+
+
+def list_commit_shas():
+    """
+    Get list of commit SHAs on ray master branch from at least 30 days ago.
```

Comment:
you can just parse `git log` of the repo locally after cloned, which might be much easier (e.g. no need to do pages).

Reply:
thank you! using `git log` is wayy easier